students/Vladimir12241/hw-1/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+    let txt = document.createElement('div') 
```

Comment:
Please use some of the auto-formatting tools to prettify you code.

https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode